task09-react/src/components/CharacterSection.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Character from './Character';
+import luffy from '../assets/img/monkeydluffy.svg';
+import ace from '../assets/img/ace.svg';
+import zoro from '../assets/img/zoro.svg';
+import nico from '../assets/img/nico.svg';
+
+class CharacterSection extends Component {
+  render() { 
+    return (
+      <article className="article article--characters">
+        <div className="article__container article__container--characters">
+          <h3 className="article__title article__title--characters">Characters</h3>
+          <a href="#" className="article__link article__link--characters">See all</a>
+        </div>
+        <div className="article__flex article__flex--characters">
+          <Character name="Monkey D. Luffy" img={luffy} alt="Monkey D. Luffy" />
+          <Character name="Portgas D. Ace" img={ace} alt="Ace" />
+          <Character name="Roronoa Zoro" img={zoro} alt="Zoro" />
+          <Character name="Nico Robin" img={nico} alt="Nico Robin" />
+        </div>
+      </article>
+    );
+  }
+}
+ 
+export default CharacterSection;
```

Comment:
Create a const object outside the component, and define in there the 4 characters with their properties, then create a loop to create the characters in the render function.
Something like: `characters.forEach((c) => {<Character name={c.name} img={c.img} .... />})`

Reply:
Thanks for the feedback. I agree. Having a loop for each character in an array makes it easy to add or remove characters without having to modify the render code.